cvat-core/src/logger-storage.ts

```diff
@@ -109,10 +139,11 @@ Object.defineProperties(LoggerStorage.prototype.log, {
                 const ignoreRule = this.ignoreRules[logType];
                 const { lastLog } = ignoreRule;
                 if (lastLog && ignoreRule.ignore(lastLog, payload)) {
-                    lastLog.payload = {
-                        ...lastLog.payload,
-                        ...payload,
-                    };
+                    lastLog.payload = ignoreRule.update(lastLog, payload);
+
+                    if (this.closeOnLog.includes(logType)) {
+                        await lastLog.close();
+                    }
```

Comment:
This probably shall be handled out of ``(logType in this.ignoreRules) {`` condition body

Reply:
Do we have any strong reasons to close log immediately after it was added?
IMHO it does not look obvious, when the log is closed implictitly. Why not to close the log from outside code after it was created?